Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,462 @@
+of the different phases; design, sanity tests, experimentation, and data analysis.
+
+\subsubsection{Design Phase}
+\label{subsubsec: design phase}
+    This will be the first phase and reappearing after the culmination of the experimentation phase.
+    Goals are to design the propulsion system, re-design the test stand, design the minimum length and aerospike nozzle
+
+    \textit{Projected End Date: Week 4 (maybe 5 if schedule slips, but no later)}
+
+\subsubsection{Build \& Preliminary Test Phase}
+\label{subsubsec: sanity test phase}
+    This phase is concerned with the building of the system. The test stand is already built. However a few revisions might be applied, then the team will need
+    to re-build to the new design. The main part of the building stage will be the construction of the propulsion system. This has not been built yet therefore, that
+    is where most of the work will be. Once the system is built, it will be tested at component-level then tested at system-level. This is to ensure there are no catastrophic
+    problems during the experiment. There will be no data collected during this phase.
+
+    \textit{Projected End Date: Week 6 (7 if fabrication takes longer than expected)}
+
+\subsubsection{Experimentation}
+\label{subsubsec:experimentation}
+    This phase is primarily concerned with the testing of the CGT\@. This includes gathering all the necessary data and experience to create the
+    polynomial approximation and to aid in the RCS phase and CREST phase of space propulsion research.
+    The testing will consist of 5 tests for each different chamber pressure. These chamber pressures will be; 80, 90, 100, 120, 140, 160, 180, and 200 Psi. There
+    will be a total of 40 tests, if time permits there can be more but this is the goal as it will serve as sufficient data for data analysis.
+     The data will be averaged for each chamber pressure and then plotted against the nozzle area ratio. Then a tenth-order polynomial
+    will be fit to the curve and used to determine a first-order approximation of the nozzle area ratio for future cold gas propulsion systems.
+
+    \textit{Projected End Date: Week 9 (10 if schedule slip occurs)}
+
+\subsubsection{Data Analysis}
+\label{subsubsec: data analysis}
+    This phase is concerned with the analysis of the data collected from the CGT testing. The polynomial approximation will be completed during this
+    phase for the minimum length nozzle. It is also the phase where the performance analysis takes place. This will inlcude the ultimate calculation of the specific impulse
+    of the thuster and if it attains the values we need to achieve the necessary velocity change for orbital decay to initiate atmospheric reentry.
+
+    \textit{Projected End Date: Week 14 or 15 (not terribly important since the team does not theoretically need to be at RIT to conduct this analysis)}
```

Comment:
Data analysis and generating the final report must be completed before the end of the semester. If the project requires more time, then it should be allocated more than one semester.

Deliverables and objectives should be clearly marked as mandatory or optional.